add contributor guidelines

danielhuppmann opened the pull request:
This PR adds a simple contributor guidelines document following the suggestion of reviewers at https://github.com/openjournals/joss-reviews/issues/1095.

> Are there clear guidelines for third parties wishing to 
 - [x] 1) Contribute to the software 
 - [x] 2) Report issues or problems with the software
 - [x] 3) Seek support

gidden commented:
Hey @danielhuppmann this looks like a great first cut. The JOSS reviewers include the following list:

```
Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
```

Do you think we cover everything there with existing docs + this PR?

See the conversation [here](https://github.com/openjournals/joss-reviews/issues/1095)

danielhuppmann commented:
Yes, I had a look at the review discussion and the suggested examples for contributor guidelines (again).
> 1) Contribute to the software
Covered by paragraph 3
> 2) Report issues or problems with the software
Covered by paragraph 2
> 3) Seek support

Well, in the current status, I would say covered by paragraph 2. We may want to start a listserv or forum at some point - now?

> Am 17.12.2018 um 17:10 schrieb Matthew Gidden <notifications@github.com>:
> 
> Hey @danielhuppmann this looks like a great first cut. The JOSS reviewers include the following list:
> 
> Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
> Do you think we cover everything there with existing docs + this PR?
> 
> See the conversation here
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

danielhuppmann commented:
added the reference to the Googlegroup in the contributor guidelines and on the documentation pages, updated the PR description to include the JOSS documentation review question (and all boxes now ticked off).

gidden commented:
Merging in so we can try to finish off Joss stuff today.
